public_dropin_environments/python3_pmml/dr_requirements.txt

```diff
@@ -1,2 +1,2 @@
-pyarrow==0.14.1
-datarobot-drum==1.9.4
+pyarrow>=0.14.1,<=3.0.0
```

Comment:
Do we want to have such a high upper constraint for `pyarrow`? I see `pyarrow==3.0.0` was released on Jan 2021 ([LINK](https://pypi.org/project/pyarrow/3.0.0/))

Reply:
This is the first version supporting py3.9. Should we use a version  that supports 3.9, or earlier can be used?

Reply:
Oh my bad forgot that you're upgrading to a version that supports python3.9. Then yes, `pyarrow==3.0.0` is the first latest version of pyarrow that supports python3.9 so let's go with that. Thanks for clarifying!